unyt/array.py

```diff
@@ -2125,6 +2125,11 @@ def dot(self, b, out=None):
             out.units = res_units
         return ret
 
+    def take(self, *args, **kwargs):
```

Comment:
Done!

Reply:
I'm still seeing `*args` and `**kwargs` in the signature, did you make a mistake ?

Reply:
Ah I changed the other one... was thinking this one just needed to pass through whatever came in but since it's public it should probably actually have the arguments.